custom_components/nordpool/__init__.py

```diff
@@ -71,9 +71,6 @@ async def _update(self, type_="today", dt=None):
             data = await spot.hourly(end_date=dt)
             if data:
                 self._data[currency][type_] = data["areas"]
-            else:
-                _LOGGER.debug("Some crap happend, retrying request later.")
```

Comment:
Perhaps a log message with `warning` would be better? It would also be good if the entire `await spot.hourly` is put under a try…except in case `spot.hourly` fails. Finally, we might want to say "Error fetching data from Nordpool, will retry later."